### PR TITLE
Make vanilla progressives for am2r and MSR too

### PR DESCRIPTION
**Pull request by Miepee**
I thought these were added when they got added for dread and echoes. apparently not.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: Chaos Options to randomly make a room dark, submerged in water or submerged in lava.
 - Added: 10 Videos added to the logic database.
+- Progressive pickups can now be configured to be placed vanilla.
```
Comment (ThanatosGit) [inline CHANGELOG.md:26]: Missing `Added`

File: CHANGELOG.md [hunk 2]
```diff
@@ -93,6 +94,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: New door types: `Access Open` and `Lightning Armor`. Access Open adds new doors to most 3-Tile high open transitions, which can then be shuffled. Lightning Armor doors can be opened with a Lightning Melee.
 - Added: Option to configure if heated rooms or lava deals constant instead of scaled damage.
 - Added: If you don't have the Baby Metroid, a configurable hint is now displayed in the textbox after collecting all DNA.
+- Progressive pickups can now be configured to be placed vanilla.
```
Comment (ThanatosGit) [inline CHANGELOG.md:97]: Same

**Review by ThanatosGit — CHANGES_REQUESTED** [review]
And please no new features on release day :p

**Comment by Miepee** [comment]
> And please no new features on release day :p

For me this is less a new feature, and more  "fix what the original PR was supposed to do".  
But almost no one plays with vanilla items so I'm fine waiting another month. (Even if they do, they still have a workaround)

**Comment by henriquegemignani** [comment]
Even if the original PR intended for this, it just means the reason won't have the feature then.

**Review by dyceron — APPROVED** [review]

**Pull request merged**